### DISCUSSION TITLE
Clarity theme for gh-pages/jekyll

### DISCUSSION THREAD

**Discussion by tenthirtyam** [vmware/clarity#6420] [Ideas] [answered]
As part of the project releases, include and maintain an example theme that can be easily consumed by gh-pages/jekyll for project sites.

**Comment by hippee-lee** [comment, marked as answer]
It sounds like the ask here is for Clarity to build and maintain a gh-pages/jekyll theme. I don't believe that the component repo is the best place for something like that. We do ship themes but they are not tied to any specific html/template structure or implementation. The themes we ship are meant to be consumed by applications or static sites built with jekyll.

IMO the best place for a gh-pages/jekyll theme would be in it's own repo maintained by gh-pages/jekyll specialists.

**Reply by tenthirtyam** [reply]
Having a companion/adjacent repository (e.g. `vmware/clarity-jekyll`) maintained to demonstrate the use **with** gh-pages/jekyll would be a good approach. 

The ability for VMware originating and community projects to use the theme based on the Clarity components could help to expand adoption and community interests, perhaps.

Ryan

**Reply by hippee-lee** [reply]
Thanks Ryan,

Something like `vmware/clarity-jekyll` would have to go through the OSPO open sourcing process and there are guidelines and requirements around projects that can be opensourced and added to the vmware org on github. Rather than creating a VMware maintained theme, IMO - creating a static site theme seems like an opportunity for someone in the community who specializes in jekyll to create and maintain it so that the theme code can stay specific and focused on the static site generator and the theme.

If you decide to do anything with this, please let us know, I'd be happy to publicize it on twitter like I did for the folks who built the Hugo theme on top of Clarity last year.

**Reply by tenthirtyam** [reply]
Yes, I'm very familiar with the process. If someone was willing to do an enablement session, I'd be keen to consider initiating the upstream contribution.
